From 038d7b6fd40937bdc142cf67a33b8a6ca64ee3cf Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 17 Nov 2017 16:38:41 +0800
Subject: [PATCH 2832/5242] MLK-16839-2: ARM64: dts: add clock source for asrc

commit  d96ff43ccfb2a9c1281b00a2ae4957b08e1d2a03 from
https://source.codeaurora.org/external/imx/linux-imx.git

add IMX8QM_ACM_AUD_CLK0_SEL and IMX8QM_ACM_AUD_CLK1_SEL for
asrc clock source. There is no clock gate for them, only
clock mux.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |    8 ++++----
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index a6170d0..91b4926 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2824,8 +2824,8 @@
 			<&clk IMX8QM_AUD_ASRC_0_MEM>,
 			<&clk IMX8QM_AUD_ACM_AUD_PLL_CLK0_CLK>,
 			<&clk IMX8QM_AUD_ACM_AUD_PLL_CLK1_CLK>,
-			<&clk IMX8QM_CLK_DUMMY>,
-			<&clk IMX8QM_CLK_DUMMY>,
+			<&clk IMX8QM_ACM_AUD_CLK0_SEL>,
+			<&clk IMX8QM_ACM_AUD_CLK1_SEL>,
 			<&clk IMX8QM_CLK_DUMMY>,
 			<&clk IMX8QM_CLK_DUMMY>,
 			<&clk IMX8QM_CLK_DUMMY>,
@@ -2864,8 +2864,8 @@
 			<&clk IMX8QM_AUD_ASRC_1_MEM>,
 			<&clk IMX8QM_AUD_ACM_AUD_PLL_CLK0_CLK>,
 			<&clk IMX8QM_AUD_ACM_AUD_PLL_CLK1_CLK>,
-			<&clk IMX8QM_CLK_DUMMY>,
-			<&clk IMX8QM_CLK_DUMMY>,
+			<&clk IMX8QM_ACM_AUD_CLK0_SEL>,
+			<&clk IMX8QM_ACM_AUD_CLK1_SEL>,
 			<&clk IMX8QM_CLK_DUMMY>,
 			<&clk IMX8QM_CLK_DUMMY>,
 			<&clk IMX8QM_CLK_DUMMY>,
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index c2275a8..4ec70d5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1974,8 +1974,8 @@
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_AUD_ACM_AUD_PLL_CLK0_CLK>,
 			<&clk IMX8QXP_AUD_ACM_AUD_PLL_CLK1_CLK>,
-			<&clk IMX8QXP_CLK_DUMMY>,
-			<&clk IMX8QXP_CLK_DUMMY>,
+			<&clk IMX8QXP_ACM_AUD_CLK0_SEL>,
+			<&clk IMX8QXP_ACM_AUD_CLK1_SEL>,
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_CLK_DUMMY>,
@@ -2014,8 +2014,8 @@
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_AUD_ACM_AUD_PLL_CLK0_CLK>,
 			<&clk IMX8QXP_AUD_ACM_AUD_PLL_CLK1_CLK>,
-			<&clk IMX8QXP_CLK_DUMMY>,
-			<&clk IMX8QXP_CLK_DUMMY>,
+			<&clk IMX8QXP_ACM_AUD_CLK0_CLK>,
+			<&clk IMX8QXP_ACM_AUD_CLK1_CLK>,
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_CLK_DUMMY>,
 			<&clk IMX8QXP_CLK_DUMMY>,
-- 
1.7.9.5

